src/query/service/src/sessions/session_privilege_mgr.rs

```diff
@@ -264,15 +272,15 @@ impl<'a> SessionPrivilegeManager for SessionPrivilegeManagerImpl<'a> {
     }
 
     #[async_backtrace::framed]
-    async fn has_ownership(&self, object: &OwnershipObject) -> Result<bool> {
+    async fn has_ownership(&self, object: &OwnershipObject, create_object: bool) -> Result<bool> {
```

Comment:
create_object -> current_role_only ?

`create_object` is bound to a concerete scenary, it might makes the semantic of this parameter decoupled from the special checks about `CREATE XXX`, but simply "only care the current role".

Reply:
Yes . check_current_role_only is better.